x-pack/legacy/plugins/infra/public/components/waffle/node_context_menu.tsx

```diff
@@ -29,102 +28,99 @@ interface Props {
   nodeType: InfraNodeType;
   isPopoverOpen: boolean;
   closePopover: () => void;
-  uiCapabilities: UICapabilities;
   popoverPosition: EuiPopoverProps['anchorPosition'];
 }
 
-export const NodeContextMenu = injectUICapabilities(
-  ({
-    options,
-    currentTime,
-    children,
-    node,
-    isPopoverOpen,
-    closePopover,
-    nodeType,
-    uiCapabilities,
-    popoverPosition,
-  }: Props) => {
-    const inventoryModel = findInventoryModel(nodeType);
-    // Due to the changing nature of the fields between APM and this UI,
-    // We need to have some exceptions until 7.0 & ECS is finalized. Reference
-    // #26620 for the details for these fields.
-    // TODO: This is tech debt, remove it after 7.0 & ECS migration.
-    const apmField = nodeType === InfraNodeType.host ? 'host.hostname' : inventoryModel.fields.id;
+export const NodeContextMenu = ({
+  options,
+  currentTime,
+  children,
+  node,
+  isPopoverOpen,
+  closePopover,
+  nodeType,
+  popoverPosition,
+}: Props) => {
+  const uiCapabilities = useKibana().services.application?.capabilities;
+  const inventoryModel = findInventoryModel(nodeType);
+  // Due to the changing nature of the fields between APM and this UI,
+  // We need to have some exceptions until 7.0 & ECS is finalized. Reference
+  // #26620 for the details for these fields.
+  // TODO: This is tech debt, remove it after 7.0 & ECS migration.
+  const apmField = nodeType === InfraNodeType.host ? 'host.hostname' : inventoryModel.fields.id;
```

Comment:
@simianhacker do you have context about this comment? Can we remove something and get rid of this comment now that we're post 7.0?